sdk/src/main/java/com/silanis/esl/sdk/examples/SigningStyleExample.java

```diff
@@ -28,5 +35,32 @@ protected void execute() {
+        SigningLogo signingLogoEn = SigningLogoBuilder.newSigningLogo()
+                .withLanguage(Locale.ENGLISH)
+                .withImage(logoEn)
+                .build();
+        signingLogos.add(signingLogoEn);
+        // Create signing logos
+        eslClient.getSigningStyleService().saveSigningLogos(signingLogos);
+        createdSigningLogos = eslClient.getSigningStyleService().getSigningLogos();
+
+        String logoFr = "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAQIAAAAoCAIAAACabWPjAAAAAXNSR0IArs4c6QAAAARnQU1BAACxjwv8YQUAAAAJcEhZcwAADsMAAA7DAcdvqGQAAAPVSURBVHhe7dy/SxthGAfw06RHGkXB3SFXUjqE/AsFm1qKUNDBXRCuIA7V4ZaMLhmsQyg0ILhniJNDbSr4LwQnA8ngHlBSG0KifX8ld+97P7yogV7z/XDUu9zrcyd9n/d93jtRAwAAAAAAIKbE14H7+3uxB/D/mpqSev60+AowwZAGAEgDAKQBAIE0AEAaACANAAikAQDSAGDC3iKftRfNrth3yiTPjxOG31lNP6rP5viubxtNW5u9KujDBuulhf0lfoLo7qbbZfVDrXF487bQEwealrXmTzZj4kDEcVyaqVqtjcrgWk4+7WX90ur13oU48GssLjEg3ZUQKs6/DW+RR9bdSLerYj+sshn8LSQxWs4cIGqF68XVTkMcPbdmZyXt7LsE+blaK4d9cUTQNlIOEOpdhYkTPZOYBmT0WrhybmQqEKcI5ezsOv2weyD9N7sikE0ZntVvkVQtOjk445xbcfrBxe32WPpTv7RzW6M79hWP1uhxrfC71KQ7AW3IXX07Yzuh4kQRZoMH6B95Vxidb89odg7oiBvPn9q1hLE5k894ViDP4ewPG79J37WvmCvwHtzb+87KPO828/yuRC0XJk4kIQ2CiS6rvTFG6p16no7u3j2j8atLB9SM/j7FP+Bi5vHCWHKATD4/2G2s6UoFn/vAZrDLO1LzNBp3dF9tI91VmDjRNIlpQGvZRXu7kcds+ewyqwEyyS3H0tYVgWxKEO0VG921SntXlBMur6cdlZgf9UJK4T6SbNqVY6l4lvx70aNpUKcLFY82LsFxogmzwQNI6aIsHkKKmdt0jCwXx7bqHSu2XLYz0IpuwRMGlsj1OVMqTsRZsTrUepce9b0SgWxKEGbpJZ0Q7PWlLFQJoV5ocFePUau7Ft/NHpvr4iTJjTRdo3u0cQmOE02YDXzw1SFRNtWCJ7SY+TVJqoWyyZ8LCcY7nZUQ3Z9y2Ko1rkFX1O6VjvyD9EtFdjlWnhkG6wrDNqnEiSvxwsSJJqSBL9GJ6Up357GFTSrxxT1+p158ognW21u23y00Dm9o3V9pj+UB/JLOHvtKV6xa/PF/PP+ZdW4+d8lthk8IhDBxIglpECCVKHo/zncvkVt+b75yBf7mwSlmHovXEcM44lVaJll80sMiv7W7vn/KU9puwFfbWWtmUM553JV4QqBp63weCBUnipAGgYzNOV4VPOH1kL7Fc0mi7w9fmQ1krflHrcXDoUWOKPMG6OpffkRL70pZftC7qjt+ByRUnMjBX6aASYTfKQJQIQ0AkAYASAMAAmkAgDQAQBoAEEgDAAAAAAAAAACgNO0vBzE5dtSK0xIAAAAASUVORK5CYII=";
+        SigningLogo signingLogoFr = SigningLogoBuilder.newSigningLogo()
+                .withLanguage(Locale.FRENCH)
+                .withImage(logoFr)
+                .build();
+        signingLogos.add(signingLogoFr);
+        // Update signing logos
+        eslClient.getSigningStyleService().saveSigningLogos(signingLogos);
+        updatedSigningLogos = eslClient.getSigningStyleService().getSigningLogos();
+
+        // Delete signing logos
+        eslClient.getSigningStyleService().saveSigningLogos(new ArrayList<SigningLogo>());
+        removedSigningLogos = eslClient.getSigningStyleService().getSigningLogos();
```

Comment:
This part has already been modified and doesn't need to be modified. Please rebase the dev branch.